nx.json

```diff
@@ -2,18 +2,15 @@
   "cli": {
     "packageManager": "bun"
   },
-  "cacheableOperations": ["build", "test", "check", "check:fix"],
   "targetDefaults": {
     "build": {
       "dependsOn": ["^build"],
       "cache": true
     },
     "check": {
-      "dependsOn": ["^check"],
       "cache": true
     },
     "check:fix": {
-      "dependsOn": ["^check:fix"],
```

Comment:
If we `check` a package, we don't necessarily want to check its dependencies. This way, if a package is updated (and thus in-scope for `lerna run check --since <ref>`), only it is checked, not its dependencies.